nautobot/extras/api/relationships.py

```diff
@@ -259,6 +256,49 @@ def to_internal_value(self, data):
         return {self.field_name: output_data}
 
 
+def api_relationships_errors(instance, relationships_data):
```

Comment:
Docstring please? It's not obvious to me without a side-by-side comparison what the difference is between `api_relationships_errors` and `get_relationships_errors`.